db/create_tables.js

```diff
@@ -0,0 +1,16 @@
+const { Shoe, Color, Size, Quantity, Shoecolor, Shoesize } = require('./schema.js');
+
+async function synchronizeModels() {
+  try {
+    await Shoe.sync();
+    await Color.sync();
+    await Size.sync();
+    await Quantity.sync();
+    await Shoecolor.sync();
+    await Shoesize.sync();
+  } catch (error) {
+    console.error(error);
+  }
+}
```

Comment:
Just an idea to maybe add a "finally" that console logs when this is complete to let the user know the tables have been added. Not a necessity.